devdox/app/repositories/repo_repository.py

```
@@ -1,12 +1,19 @@
-from typing import Any, Coroutine, List
+from abc import abstractmethod
+from typing import Any, List, Protocol
 
-from tortoise.exceptions import IntegrityError
-
-from app.exceptions.custom_exceptions import BadRequest
 from models import Repo
 
+class IRepoStore(Protocol):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)